docs/UserGuide.md

```diff
@@ -180,6 +182,41 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Creating an event: `create`
+
+Creates a new event in the address book.
+
+Format: `create t/EVENT_TITLE d/DATE t/TIME p/PURPOSE`
+
+* The compulsory parameters are: event title (t), date (d), time (t) and purpose (p)
+
+Examples:
+* `create t/Shoe Sale 30% d/30-05-2022 t/11:00 p/Discount on all shoes for up to 30%`
+* `create t/Banana Discount 10% d/20-04-2022 t/14:00 p/10% discount on all bananas`
+
+### Removing an event: `remove`
+
+Removes an existing event in the address book.
+
+Format: `remove INDEX`
+
+* Removes the event at the specified INDEX. 
+* The index refers to the index number shown in the displayed person list 
```

Comment:
I think you are referring to the event list here, not person list.

```suggestion
* The index refers to the index number shown in the displayed event list 
```

Reply:
Have made this change!